### PR TITLE
Add JSON schema for Package and Vulnerability metadata

### PR DESCRIPTION
**Pull request by keshav-space**
- Resolves https://github.com/nexB/federatedcode/issues/10

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -48,6 +48,7 @@ install_requires =
     django-rest-framework>=0.1.0
     djangorestframework>=3.14.0
     django-environ>=0.10.0
+    django-ninja==1.2.1
```
Comment (pombredanne) [inline setup.cfg:51]: add pydantic and on put a minimum version for this.

**Review by pombredanne — COMMENTED** [review]
LGTM! I have just a minor nit wrt to versions of dependencies

File: setup.cfg (hunk 1 above)
Comment (keshav-space) [reply setup.cfg]: Done!

**Comment by pombredanne** [comment]
@keshav-space ignore this... config error on my part. Sorry for the churn!

**Pull request merged**